bcipy/acquisition/multimodal.py

```diff
@@ -1,12 +1,44 @@
 """Functionality for managing multiple devices."""
 import logging
-from typing import Dict, List, Optional
-from bcipy.acquisition.protocols.lsl.lsl_client import LslAcquisitionClient
+from typing import Any, Dict, List, Optional
+
 from bcipy.acquisition.devices import DeviceSpec
+from bcipy.acquisition.exceptions import (InsufficientDataException,
+                                          UnsupportedContentType)
+from bcipy.acquisition.protocols.lsl.lsl_client import LslAcquisitionClient
+from bcipy.acquisition.record import Record
+from bcipy.helpers.system_utils import AutoNumberEnum
 
 log = logging.getLogger(__name__)
 
 
+class ContentType(AutoNumberEnum):
+    """Enum of supported acquisition device (LSL) content types. Allows for
+    case-insensitive matching, as well as synonyms for some types.
+
+    >>> ContentType(1) == ContentType.EEG
+    True
+    >>> ContentType('Eeg') == ContentType.EEG
+    True
+    """
+
+    def __init__(self, synonyms: List[str]):
+        self.synonyms = synonyms
+
+    EEG = []
```

Comment:
As discussed in the morning, we can make the distinction between ERP and VEP here.

Reply:
EEG here refers to the content_type that is sent by LSL. So even when we are collecting VEP the source of that data provided by LSL will be EEG. Where we distinguish between the two might be in the EvidenceType. But I think I would like to revisit that when we are implementing other aspects of VEP. Otherwise we will have an evidence type that's not used yet.